<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProgWindow</class>
 <widget class="QMainWindow" name="ProgWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>350</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>380</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>380</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>380</width>
     <height>307</height>
    </size>
   </property>
   <widget class="QListWidget" name="commandList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>360</width>
      <height>290</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>360</width>
      <height>290</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1680</width>
      <height>720</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">selection-background-color: rgb(159, 181, 255);</string>
    </property>
    <property name="autoScrollMargin">
     <number>1</number>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="showDropIndicator" stdset="0">
     <bool>true</bool>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>true</bool>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::InternalMove</enum>
    </property>
    <property name="defaultDropAction">
     <enum>Qt::MoveAction</enum>
    </property>
    <property name="alternatingRowColors">
     <bool>false</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::ExtendedSelection</enum>
    </property>
    <property name="uniformItemSizes">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>380</width>
     <height>22</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Open"/>
    <addaction name="actionSave"/>
    <addaction name="actionOptions"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionNew_Command"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionDuplicate"/>
    <addaction name="actionDelete"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew_Open">
   <property name="text">
    <string>New/Open</string>
   </property>
   <property name="toolTip">
    <string>Create a new program or open an existing one</string>
   </property>
   <property name="statusTip">
    <string>Create a new program or open an existing one</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save the current state of the program</string>
   </property>
   <property name="statusTip">
    <string>Save the current state of the program</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit macomator</string>
   </property>
   <property name="statusTip">
    <string>Quit macomator</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="toolTip">
    <string>Open the settings menu</string>
   </property>
   <property name="statusTip">
    <string>Open the settings menu</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </action>
  <action name="actionNew_Command">
   <property name="text">
    <string>Add Command</string>
   </property>
   <property name="statusTip">
    <string>Create a new command</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="statusTip">
    <string>Copy the currently selected items</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="statusTip">
    <string>Removes the selected element(s)</string>
   </property>
   <property name="shortcut">
    <string>Backspace</string>
   </property>
  </action>
  <action name="actionDuplicate">
   <property name="text">
    <string>Duplicate</string>
   </property>
   <property name="statusTip">
    <string>Duplicates the selected element(s)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
